src/bugreporter/odemis_bugreporter.py

```diff
@@ -256,18 +256,17 @@ def compress_files(self):
                 models = glob(os.path.join(odemis_config['CONFIGPATH'], '*/*.odm.yaml'))
             files.extend(models)
 
-            # FIXME: the following two are not working properly if glob doesn't sort by time,
-            # which it probably doesn't do.
-
             # Add the latest overlay-report if it's possibly related (ie, less than a day old)
-            overlay_reps = glob(os.path.join(home_dir, 'odemis-overlay-report', '*'))
+            overlay_reps = glob(os.path.join(home_dir, 'odemis-overlay-report/*'))
+            # glob returns list of folders sorted by their names. The folder name corresponds
```

Comment:
No, it's not sorted by names. Most likely it's sorted by order it's on the filesystem (but you shouldn't even rely on this). It's an arbitrary order, but indeed, it could well be more or less sorted by time in the common cases. 
Anyway, I'd suggest to make entirely sure by sorting it. It's pretty easy, just add:
`overlay_reps.sort(key=os.path.getmtime)`